custom/icds_reports/reports/demographics_data.py

```diff
@@ -154,28 +179,35 @@ def get_data_for(query_class, filters):
                     'format': 'percent_and_div',
                     'frequency': frequency,
                     'redirect': 'demographics/lactating_enrolled_women'
-                },
-                {
-                    'label': _(ADOLESCENT_GIRLS_ENROLLED_FOR_ANGANWADI_SERVICES),
-                    'help_text': percent_adolescent_girls_enrolled_help_text(),
-                    'percent': percent_diff(
-                        'person_adolescent',
-                        data,
-                        prev_data,
-                        'person_adolescent_all'
-                    ),
-                    'color': get_color_with_green_positive(percent_diff(
-                        'person_adolescent',
-                        data,
-                        prev_data,
-                        'person_adolescent_all'
-                    )),
-                    'value': get_value(data, 'person_adolescent'),
-                    'all': get_value(data, 'person_adolescent_all'),
-                    'format': 'percent_and_div',
-                    'frequency': frequency,
-                    'redirect': 'demographics/adolescent_girls'
                 }
+
             ]
         ]
     }
+
```

Comment:
This changes the program summary page tile
![program_summary2](https://user-images.githubusercontent.com/21310920/71279286-c703fc00-237e-11ea-8a3b-08f7243f149e.png)